src/components/content/common/doc/CredentialApiDoc.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { LinkOutlined } from '@ant-design/icons';
+import '../../../../styles/app.css';
+import { CredentialsManagementService, CredentialVariables, Link } from '../../../../xpanse-api/generated';
+
+export function CredentialApiDoc({
```

Comment:
since the method is used only by credential components, we can move this to `credential`  folder instead of common folder.